src/app/pages/send-tokens/components/send-form-inner.tsx

```diff
@@ -59,60 +90,93 @@ export function SendFormInner(props: SendFormInnerProps) {
-          isSponsored={isSponsored}
-        />
-      ) : (
-        <LoadingRectangle height="32px" width="100%" />
-      )}
-      <Box mt="auto">
-        {assetError && (
-          <ErrorLabel mb="base">
-            <Text textStyle="caption">{assetError}</Text>
-          </ErrorLabel>
+    <>
+      <Stack
+        maxWidth={CENTERED_FULL_PAGE_MAX_WIDTH}
+        mt="loose"
+        px={['loose', 'base-loose']}
+        spacing="loose"
+        width="100%"
+      >
+        <AssetSearch onSelectAsset={onSelectAsset} />
+        <Suspense fallback={<></>}>
+          <AmountField error={errors.amount} value={values.amount || 0} />
+        </Suspense>
+        <RecipientField error={errors.recipient} value={values.recipient} />
+        {selectedAsset?.hasMemo && <MemoField value={values.memo} error={errors.memo} />}
+        {selectedAsset?.hasMemo && symbol && <SendFormMemoWarning symbol={symbol} />}
+        {feeEstimations.estimates.length ? (
+          <FeeRow
+            feeEstimations={feeEstimations.estimates}
+            feeFieldName="fee"
+            feeTypeFieldName="feeType"
+            isSponsored={isSponsored}
+          />
+        ) : (
+          <LoadingRectangle height="32px" width="100%" />
         )}
-        <PrimaryButton
-          data-testid={SendFormSelectors.BtnPreviewSendTx}
-          isDisabled={!hasValues}
-          onClick={onSubmit}
-          width="100%"
-        >
-          Preview
-        </PrimaryButton>
-      </Box>
-      <Box mb={['loose', 'unset']}>
-        <ShowEditNonceAction />
-      </Box>
-    </Stack>
+        <Box mt="auto">
+          {assetError && (
+            <ErrorLabel mb="base">
+              <Text textStyle="caption">{assetError}</Text>
+            </ErrorLabel>
+          )}
+          <PrimaryButton
+            data-testid={SendFormSelectors.BtnPreviewSendTx}
+            isDisabled={!hasValues}
+            onClick={onSubmit}
+            width="100%"
+          >
+            Preview
+          </PrimaryButton>
+        </Box>
+        <Box mb={['loose', 'unset']}>
+          <ShowEditNonceAction />
+        </Box>
+      </Stack>
+      {whenWallet({
+        ledger: <Outlet />,
+        software: (
+          <SendTokensSoftwareConfirmDrawer
+            isShowing={isShowing && !showEditNonce}
+            onClose={onConfirmDrawerClose}
+            onUserSelectBroadcastTransaction={async (
+              transaction: StacksTransaction | undefined
+            ) => {
+              if (!transaction) return;
+              const signedTx = signSoftwareWalletTx(transaction);
+              if (!signedTx) return;
+              await onBroadcastTransaction(signedTx);
+              void analytics.track('submit_fee_for_transaction', {
+                calculation: feeEstimations.calculation,
+                fee: values.fee,
+                type: values.feeType,
+              });
+            }}
+          />
+        ),
+      })}
```

Comment:
Why did we need to move this to the inner component?

Reply:
I needed to move loading fee estimations into the inner component and feeEstimations are needed here.

Reply:
I can go back and look at this today to see if there is a better solution.